### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (div)

### ISSUE THREAD

**Issue by shalinijoshi19** [microsoft/AdaptiveCards#3576]
#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.3/color-contrast?application=msftAI))
#### Target application
[Schema Explorer | Adaptive Cards](https://adaptivecards.io/explorer/)
#### Element path
button[aria-label="OK"] > div
#### Snippet
`<div style="overflow: hidden; text-overflow: ellipsis; white-space: nowrap;">OK</div>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 4.49 (foreground color: #0078d7, background color: #ffffff, font size: 10.5pt (14px), font weight: bold). Expected contrast ratio of 4.5:1
#### Environment
Microsoft Edge version 79.0.309.11
This accessibility issue was found using Accessibility Insights for Web 2.10.3 (axe-core 3.3.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
